assets/css/main.css

```diff
@@ -16,51 +16,51 @@
  */
 /* Default values */
 :root {
-  --c-white: white;
-  --c-lightGray: #f6f7f8;
-  --c-mediumGray: #757575;
-  --c-darkGray: #353e48;
-  --c-coolGrayLight: #e4e9ed;
+  --c-white: rgb(255, 255, 255);
```

Comment:
any idea where these changes came from?

Reply:
node-sass upgrade. the rgb values are correct. not sure the exact change in node-sass, but perhaps another one of their bug fixes that has a side-effect.